res/values-pt-rBR/strings.xml

```diff
@@ -958,7 +958,7 @@ Regras do Nebulous\n
 7. Não crie Jogos Personalizados/Chats com nomes inapropriados.\n
 8. Obedeça os Moderadores.\n
 9. Não aproveite-se de Bugs no jogo.\n
-10. Não dê ban em muitos jogadores do Clã sem a autorização do líder.\n
+10. Evite dar ban em muitos jogadores do Clã sem a autorização do líder.\n
```

Comment:
"Não dê" is an imperative form, so previous translation was already correct, even though your change is OK.